manifest.go

```diff
@@ -43,3 +45,16 @@ func LoadManifest() *manifest {
 
 	return m
 }
+
+func CreateManifest(name string, author string, ver string) *manifest {
```

Comment:
You need to export manifest I think. Also can you rename it to `NewManifest?`

Reply:
Don't need to export it in this case.